server/src/main/java/org/opensearch/search/aggregations/bucket/BucketsAggregator.java

```diff
@@ -129,6 +129,15 @@ public final void collectExistingBucket(LeafBucketCollector subCollector, int do
         subCollector.collect(doc, bucketOrd);
     }
 
+    public final void collectStarTreeBucket(long docCount, long bucketOrd)
+        throws IOException {
+        if (docCounts.increment(bucketOrd, docCount) == docCount) {
+            multiBucketConsumer.accept(0);
+        }
+        // Only collect own bucket & not sub-aggregator buckets
+        // subCollector.collectStarEntry(entryBit, bucketOrd);
```

Comment:
This line won't exist, right?

Could we also do this by calling `collectExistingBucket` with `LeafBucketCollector.NO_OP_COLLECTOR`?